ci/plugins/cloudtest/hooks/post-command

```diff
@@ -17,6 +17,51 @@ kubectl() {
     bin/ci-builder run stable kubectl --context=kind-cloudtest "$@"
 }
 
+export_cov() {
+    bin/ci-builder run stable rust-cov export \
+      --ignore-filename-regex=.cargo/ \
+      --ignore-filename-regex=target/release/ \
+      --ignore-filename-regex=/cargo/ \
+      --ignore-filename-regex=/mnt/build/ \
+      --ignore-filename-regex=/rustc/ \
+      --format=lcov "$1" --instr-profile=coverage/"$BUILDKITE_JOB_ID".profdata src/ \
+      > coverage/"$BUILDKITE_JOB_ID"-"$(basename "$1")".lcov
+}
+
+if [ -n "${CI_COVERAGE_ENABLED:-}" ]; then
```

Comment:
Minor: What about simplifying this to
```
if [ "${CI_COVERAGE_ENABLED}" = "true" ]; then
```
(or similar) to improve readability?

Reply:
We have set shell script to fail when you reference an unused variable, thus this would cause an error.

Reply:
Good to know, thanks!